contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -528,7 +528,8 @@ pltsql_pre_parse_analyze(ParseState *pstate, RawStmt *parseTree)
 				 */
 				if (relid == sysdatabases_oid ||
 					relid == namespace_ext_oid ||
-					relid == bbf_view_def_oid)
+					relid == bbf_view_def_oid ||
+					relid == bbf_schema_perms_oid)
```

Comment:
is there a correlation between adding an entry here vs doing config dump for catalog table? It seems we only do it for tables which we do not config dump?

Reply:
This is not needed. Removing it.